src/pkg/logs/logrus.go

```diff
@@ -5,9 +5,13 @@ import "github.com/defang-io/defang/src/pkg"
 func IsLogrusError(message string) bool {
 	// Logrus's TextFormatter prefixes messages with the uppercase log level, optionally truncated and/or in color
 	switch message[:pkg.Min(len(message), 4)] {
-	case "WARN", "ERRO", "FATA", "PANI", "\x1b[31", "\x1b[33": // red or yellow
+	case "ERRO", "FATA", "PANI", "\x1b[31": // red
 		return true // always show
-	case "", "INFO", "TRAC", "DEBU", "\x1b[36", "\x1b[37": // blue or gray
+	case "WARN", "\x1b[33": // yellow
```

Comment:
Would this mask any existing/future warning messages?

Reply:
It's currently only used for filtering Kaniko logs